README.md

```diff
@@ -61,53 +62,107 @@ Versioning is not supported at this time.
 
 # Setting it up <a id="Interesting_Part"/>
 
-## Compilation
+## Building
+
+Building requires the following:
+- Docker
+- irods-dev (custom build - irods/irods sha: 6b14b65)
+- irods-runtime (custom build - irods/irods sha: 6b14b65)
```

Comment:
Perhaps specify that this is the earliest sha?

Reply:
I'm going to remove these and add a statement which links to the earliest commit where this works.

Reply:
Done.

See https://github.com/irods/irods_client_s3_cpp/pull/12#discussion_r1291579818 for updated wording.